v1_19/src/main/java/com/denizenscript/denizen/nms/v1_19/impl/BiomeNMSImpl.java

```diff
@@ -130,6 +135,24 @@ public void setPrecipitation(DownfallType type) { // TODO: 1.19.4: This is no lo
         setClimate(nmsType, getTemperature(), getTemperatureModifier(), getHumidity());*/
     }
 
+    @Override
+    public void setFoliageColor(int color) {
+        try {
+            BiomeSpecialEffects.Builder builder = (BiomeSpecialEffects.Builder) BIOMESPECIALEFFECTS_BUILDER_CONSTRUCTOR.invoke();
+            // fogColor, waterColor, waterFogColor, and skyColor are needed for the Builder class.
+            builder.fogColor(biomeHolder.value().getFogColor())
+                    .waterColor(biomeHolder.value().getWaterColor())
+                    .waterFogColor(biomeHolder.value().getWaterFogColor())
+                    .skyColor(biomeHolder.value().getSkyColor())
+                    .foliageColorOverride(color);
+            BiomeSpecialEffects effects = builder.build();

```

Comment:
Won't that loose other values stored on `BiomeSpecialEffects` (I.e. background sounds, particles, etc.)?
Can just copy them all over, although maybe consider just setting the `foliageColorOverride` field on the biome's existing `BiomeSpecialEffects` object - there is a public getter for it and it'd save having to copy everything over.